src/main/webapp/js/instructorFeedbackResults.js

```diff
@@ -291,7 +291,7 @@ $(document).ready(function() {
         toggleCollapse(this, panels);
     });
 
-    $('a[id^="collapse-panels-button-team-"]').on('click', function() {
+    $('.panel.panel-success').on('click', 'a[id^="collapse-panels-button-team-"]', function() {
```

Comment:
Just to check, should line 279 (`$('a[id^="collapse-panels-button-section-"]').on('click',`) be modified too?